src/components/qas/questions/qaCalculator/qaCalculatorUtilities.js

```diff
@@ -51,6 +51,63 @@ export const calculateKineticEnergy = ({ mass, velocity }) =>
 
 export const calculateMass = ({ density, volume }) => density * volume;
 
+export const calculateDestruction = () => {};
+
+export const calculateMinimumSafe = () => {};
+
```

Comment:
yes, you're right, we'll have to figure out what constitutes these definitions next.  Creating a ticket to update the Impact Calc with this info.  I think is as straightforward as us just deciding based on the damage descriptions what we (i.e. Lauren and Ardis) agree to be the "Area of Destruction" and "Minimum Safe Distance".

Reply:
calculating those values current eludes me, at the moment, but I'm sure as we approach this work it will become more clear.